integrationtests/tests/tests.rs

```diff
@@ -844,3 +844,49 @@ async fn rejoin_consensus_threshold_peers() {
     // confirm that the entire federation can rejoin at an epoch
     timeout(Duration::from_secs(15), rejoin).await.unwrap();
 }
+
+//TODO: remove this test the first time the adapter is used organically
```

Comment:
When will be the first time we plan to use it for a real test?

Reply:
I wanted to use it but when I did I noticed my work (gateway retry outgoing payments) was completely and I think it will take a while. I think @okjodom also wanted to use it in his last [PR](https://github.com/fedimint/fedimint/pull/602#issuecomment-1257151246)